pkg/auth/handler/webapp/authflowv2/noproject_sso_callback.go

```diff
@@ -0,0 +1,26 @@
+package authflowv2
+
+import (
+	"net/http"
+
+	handlerwebapp "github.com/authgear/authgear-server/pkg/auth/handler/webapp"
+	"github.com/authgear/authgear-server/pkg/util/httproute"
+	"github.com/authgear/authgear-server/pkg/util/template"
+)
+
+func ConfigureNoProjectSSOCallbackRoute(route httproute.Route) httproute.Route {
+	return route.
+		WithMethods("GET").
+		WithPathPattern("/noproject/sso/callback")
```

Comment:
This endpoint should support OPTIONS and POST method, due to `response_mode=form_post`.

Reply:
Saw this was fixed in a later commit.